php/post-types/class-block-post.php

```diff
@@ -598,7 +648,8 @@ class="regular-text"
 								id="block-fields-edit-name-input_<?php echo esc_attr( $uid ); ?>"
 								class="regular-text"
 								value="<?php echo esc_attr( $field->name ); ?>"
-								data-sync="block-fields-name-code_<?php echo esc_attr( $uid ); ?>" />
+								data-sync="block-fields-name-code_<?php echo esc_attr( $uid ); ?>"
+								<?php echo $is_field_disabled ? 'readonly="readonly"' : ''; ?> />
```

Comment:
You'll probably aware of this, but using `readonly` instead of `disabled` means that it will be in the `POST` request. Having the name of the control in the request is important.

Reply:
Makes sense. Maybe we could add some CSS along the lines of:
```
.block-fields-edit input[readonly="readonly"],
.block-fields-edit textarea[readonly="readonly"] {
    color: #999;
}
```